src/components/JoinSection/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { FC } from 'react';
```

Comment:
Super picky, but I think we generally import React and then use `React.FC`. Can we change it here so we keep it consistent?

Reply:
yes i forgot that xd